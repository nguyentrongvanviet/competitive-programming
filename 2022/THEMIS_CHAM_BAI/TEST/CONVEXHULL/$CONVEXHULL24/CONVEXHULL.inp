72
-597078567 264562101
294232976 -453883535
-511174757 483047010
-227501053 -40531473
526900588 -688741306
464097483 59509342
508837743 609881887
225561559 -264330619
-204507933 70561796
528041641 -752696766
-685602044 -170432215
6063988 -118870218
179129407 7183029
23451600 217164002
-413457389 -89949810
-889817233 -341897897
272573646 248717206
-370607885 250399194
356061244 348850440
5037722 -111636933
197887103 61931423
-182126835 -62010876
-408012646 -839986617
89697462 121400973
-166569243 184488345
693477430 24727159
176419222 729877562
700285825 81749458
520796514 418824624
-1040532 -394145795
219606329 -248761404
-403145006 233226417
819023588 47106138
210610271 -396096858
-53332570 -24489307
-605548495 -194470721
263005994 354326637
-824266925 -158920980
-496998643 29169791
-346744303 195260595
-339370760 -457160570
924944664 -124983155
-416628369 109845772
770546226 -458095748
-2184054 -432087454
629105490 144837133
-675841919 240496389
602959392 716001944
415645481 -195310362
478958685 -255739968
-78680605 -120790097
205898594 -694507358
195257549 -380294850
-160698321 -272727575
-124419254 152494710
-249315133 237252598
-170170437 99847545
53448972 627516327
-419262457 -411705861
194343163 102878009
426956352 -172637328
-855834890 467021311
300740098 94771991
-134890231 80199970
-990709 1223941
292119307 -447331634
87145458 -358865733
-292352233 -784987228
29345478 -5229759
-9396453 5364962
-900023917 33696135
4646833 -11448505
