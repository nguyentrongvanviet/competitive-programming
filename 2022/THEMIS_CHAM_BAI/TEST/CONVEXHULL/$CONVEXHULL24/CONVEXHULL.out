9
2255862725240499248.5
-408012646 -839986617
528041641 -752696766
770546226 -458095748
924944664 -124983155
602959392 716001944
176419222 729877562
-855834890 467021311
-900023917 33696135
-889817233 -341897897
